data-pipeline/src/trace_exporter/error.rs

```diff
@@ -157,6 +157,10 @@ impl RequestError {
     pub fn status(&self) -> StatusCode {
         self.code
     }
+
+    pub fn msg(&self) -> &str {
+        &self.msg
+    }
```

Comment:
I know it's the most trivial of functions, but since it's public we should probably have a unit test.